pyathena/connection.py

```diff
@@ -136,8 +138,15 @@ def __init__(
                 profile_name=self.profile_name,
                 **self._session_kwargs,
             )
+
+        self.config: Optional[config.Config] = config
+        self.config.user_agent_extra = (
+            "PyAthena/"
+            + pyathena.__version__
+            + (" " + config.user_agent_extra if config.user_agent_extra else "")
+        )
```

Comment:
```suggestion
        self.config.user_agent_extra = (
            f"PyAthena/{pyathena.__version__}"
            f"{' ' + config.user_agent_extra if config.user_agent_extra else ''}"
        )
```

Reply:
I changed to use f-string but I'm using the self.config variable in the ternary operator. Otherwise, if parameter is None I would get:
`AttributeError: 'NoneType' object has no attribute 'user_agent_extra'`

Reply:
Thanks for the update. The AttributeError should no longer occur with the following changes.
https://github.com/laughingman7743/PyAthena/pull/383#discussion_r1032624262